e2e/pages/caseList/caseList.js

```diff
@@ -9,4 +9,9 @@ module.exports = {
     const caseLink = locate('a').withAttr({href: href});
     I.click(caseLink);
   },
+
+  changeStateFilter(desiredState) {
```

Comment:
This is not really needed now. Shall we delete it or keep it in case we need it in the future?

Reply:
I was thinking it would be used in the future when play some stories around filtering

Reply:
Alright, happy to keep it.